### PR TITLE
Adding ga events to thank you page video

### PR DESCRIPTION
**Pull request by vojtechmzl**
Hi @beccaklam @cadecairos, here is the patch that adds Google Analytics events to the thank you videos. Please let me know if you have any questions or suggestions. Thanks!

### CODE REVIEW

File: src/pages/jan-thank-you.js [hunk 1]
```diff
@@ -34,8 +34,20 @@ var ThankYou = React.createClass({
...
-    analytics();
+  videoGaEvent(props) {
+    props.category = "Thank You Video";
+    ga.event(props);
+  },
+  componentDidMount() {
+    var video = this.refs.video;
+    var videoId = pageDataSet[this.props.route.pageType];
+
+    video.addEventListener("play", (e) =>{
```
Comment (cadecairos) [inline src/pages/jan-thank-you.js]: Can you add a space after the `=>`

File: src/pages/jan-thank-you.js [hunk 2]
```diff
@@ -34,8 +34,20 @@ var ThankYou = React.createClass({
...
+      this.videoGaEvent({action: "Video started",label: videoId.thankYouSentenceId});
+    });
+    video.addEventListener("ended", (e) => {
+      this.videoGaEvent({action: "Video ended",label: videoId.thankYouSentenceId});
```
Comment (cadecairos) [inline src/pages/jan-thank-you.js]: Please add a space between the action and label attributes

Comment (cadecairos) [inline src/pages/jan-thank-you.js]: Line 46 need the same fix

Comment (vojtechmzl) [inline src/pages/jan-thank-you.js]: Thanks for the flag, Chris. Just updated.

File: src/pages/jan-thank-you.js (hunk 1 above)
Comment (vojtechmzl) [inline src/pages/jan-thank-you.js]: Thanks for the flag, just updated.

**Pull request merged**